src/features/Main/main.js

```diff
@@ -0,0 +1,154 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+import { useSelector, useDispatch } from 'react-redux';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import SlideshowWithPagination from 'react-slideshow-with-pagination';
+import { fetchClasses } from './mainSlice';
+
+const Section = styled.section`
+
+.item-card{
+  width: 500px;
+  margin-left:10px;
+ 
+}
+
+.item-wrapper {
+  display: flex;
+  height: 480px;
+  overflow: hidden;
+}
+
+.item-wrapper2 {
+    display: flex;
+    height: 480px; 
+    overflow: hidden;
+ }
+
+.item-photo-wrapper {
+  margin-top:100px;
+  margin-left:40px;
+  height: 200px;
+  width: 200px;
+  align-items: center;
+}
+
+.item-photo {
+  width: 200px;
+  height: 200px;
+  border-radius: 50%;
+  object-fit: cover;
+}
+
+.item-details {
+  height: 50%;
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-left:40px
+  text-align: center;
+}
```

Comment:
if you look at the `items details` you can see how it's tilted more to the right If you can fix that, that would be nice
![image](https://user-images.githubusercontent.com/103601335/231345868-359cefcf-ea42-46e9-b39b-6c50bd20be43.png)

Reply:
@Cluab thank you I will make the changes, the code was not done I haven't assigned a reviewer, I just pushed it from my local repo because It was late while I was working on it  the review you gave is not on the final product, I was hoping to talk with you guy's on the collaboration session.

Reply:
@frnega Okay sure thing keep in mind that by the end of the day, we are going to submit a code review so everything should be ready at most on the second session